[2025-10-11T15:04:08.264+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-10-11T15:04:08.278+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: read_sqlite_db.read_customers manual__2025-10-11T15:04:05.393604+00:00 [queued]>
[2025-10-11T15:04:08.288+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: read_sqlite_db.read_customers manual__2025-10-11T15:04:05.393604+00:00 [queued]>
[2025-10-11T15:04:08.288+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-10-11T15:04:08.330+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): read_customers> on 2025-10-11 15:04:05.393604+00:00
[2025-10-11T15:04:08.333+0000] {standard_task_runner.py:72} INFO - Started process 66 to run task
[2025-10-11T15:04:08.336+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'read_sqlite_db', 'read_customers', 'manual__2025-10-11T15:04:05.393604+00:00', '--job-id', '15', '--raw', '--subdir', 'DAGS_FOLDER/read_sqlite_db.py', '--cfg-path', '/tmp/tmpjzo8gyjk']
[2025-10-11T15:04:08.338+0000] {standard_task_runner.py:105} INFO - Job 15: Subtask read_customers
[2025-10-11T15:04:08.394+0000] {logging_mixin.py:190} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/settings.py:209 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-10-11T15:04:08.474+0000] {task_command.py:467} INFO - Running <TaskInstance: read_sqlite_db.read_customers manual__2025-10-11T15:04:05.393604+00:00 [running]> on host 5538f3f005ec
[2025-10-11T15:04:08.658+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='read_sqlite_db' AIRFLOW_CTX_TASK_ID='read_customers' AIRFLOW_CTX_EXECUTION_DATE='2025-10-11T15:04:05.393604+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-10-11T15:04:05.393604+00:00'
[2025-10-11T15:04:08.661+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-10-11T15:04:08.689+0000] {logging_mixin.py:190} INFO - âœ… Retrieved data from SQLite:
[2025-10-11T15:04:08.690+0000] {logging_mixin.py:190} INFO - (1, 'Alice2')
[2025-10-11T15:04:08.691+0000] {logging_mixin.py:190} INFO - (2, 'Bob2')
[2025-10-11T15:04:08.692+0000] {logging_mixin.py:190} INFO - (3, 'Charlie2')
[2025-10-11T15:04:08.692+0000] {python.py:240} INFO - Done. Returned value was: [(1, 'Alice2'), (2, 'Bob2'), (3, 'Charlie2')]
[2025-10-11T15:04:08.730+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-10-11T15:04:08.731+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=read_sqlite_db, task_id=read_customers, run_id=manual__2025-10-11T15:04:05.393604+00:00, execution_date=20251011T150405, start_date=20251011T150408, end_date=20251011T150408
[2025-10-11T15:04:08.792+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-10-11T15:04:08.818+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-10-11T15:04:08.821+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
